MaterialLib/MPL/Properties/CapillaryPressureSaturation/SaturationVanGenuchten.h

```diff
@@ -15,20 +15,35 @@ namespace MaterialPropertyLib
 class Medium;
 class Phase;
 class Component;
-
-/// The van Genuchten soil characteristics function.
-///
-/// This property must be a medium property, it computes the saturation of the
-/// wetting phase as function of capillary pressure.
+/**
+ *   \brief van Genuchten water retention model
+ * This property must be a medium property, it computes the saturation of the
+ * wetting phase as function of capillary pressure.
+ *
+ *   \f[p_c=p_b (S_e^{-1/m}-1)^{1-m}\f]
```

Comment:
Shouldn't it be other way around S(p_c)?

Reply:
Added a formula of S(pc) by amending the last commit.

Reply:
The implementation is slightly different. If I didn't make a mistake in writing this down, it is
Seff(p) = ((p^n + 1)^(-m), where p is pc/pb, and n is 1/(1-m). The difference is in the exponent (here n), and you have 1/m.
At the end there is clamping of S between Sr and Smax. In the beginning too, if p_c <= 0, Smax is returned.

In the documentation you introduce n, which is not explained.

Reply:
For future PR's I recommend to separate different topics in the commits, like documentation is one commit, renaming is another. Makes reviewers' life easier.

Reply:
Corrected. See the screen shot:
![f](https://user-images.githubusercontent.com/1343839/77561274-c4093200-6ebe-11ea-9eef-45486261f460.png)

Reply:
1.
> At the end there is clamping of S between Sr and Smax. In the beginning too, if p_c <= 0, Smax is returned.

2.
> In the documentation you introduce n, which is not explained.

..., and actually not used.

3.
Additionally, m can not be 1, and it is tested to be in (0,1) open interval.

Reply:
Changed as:
![f](https://user-images.githubusercontent.com/1343839/77627476-43dadf00-6f47-11ea-9f8c-5ced00256954.png)

Reply:
Also corrected: S(pc) = S_max if pc<0

Reply:
Nice. This looks much closer to the actual implementation.

Reply:
Could you explain what \alpha is?

Reply:
\alpha is an alternative  parameter of p_b of the model. For some cases, \alpha instead of p_b is given, p_b can be calculated by that formula.

Reply:
You have a reference? In the original vanGenuchten1980 paper I don't see such an expression.
Actually I also don't see the entry pressure there too.

Reply:
See Olaf'book or
https://mooseframework.inl.gov/modules/porous_flow/capillary_pressure.html